internal/research/opentelemetry/deployment-metrics.md

```diff
@@ -0,0 +1,59 @@
+# OpenTelemetry Collector Usage for Metrics
+
+OpenTelemetry is a project to handle telemetry data (logs, metrics and traces) in a vendor-neutral way. It provides SDKs to instrument applications, APIs and processing tools (collector).
+
+This document describes different ways to deploy the OpenTelemetry collector to handle metrics and findings from a PoC. To demonstrate the functionality of OpenTelemetry, we deployed an application that exports metrics via the Prometheus `/metrics` HTTP path.
+
+## Collector Services
+
+The collector consists of telemetry receivers, processors and exporters, which can be combined to processing pipelines in services. For example, the following configuration pulls metrics by scraping a Prometheus compatible HTTP endpoint and can receive metrics that are pushed by the OTLP protocol. Then, it forwards the metrics to another central collector via the OTLP protocol. Scraping Prometheus metrics requires a [Prometheus scrape config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config) for the receiver.
```

Comment:
```suggestion
The Collector consists of telemetry receivers, processors and exporters, which can be combined to processing pipelines in services. For example, the following configuration pulls metrics by scraping a Prometheus compatible HTTP endpoint and can receive metrics that are pushed by the OTLP protocol. Then, it forwards the metrics to another central collector via the OTLP protocol. Scraping Prometheus metrics requires a [Prometheus scrape config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config) for the receiver.
```

...not sure whether the "other central collector" is the same thing as the capitalized OpenTelemetry Collector?